skyportal/models.py

```diff
@@ -1122,6 +1122,8 @@ class Allocation(Base):
         back_populates='allocations',
         doc="The Instrument the allocation is associated with.",
     )
+    username = sa.Column(sa.String, doc="The username of the PI's.")
```

Comment:
Store as encrypted JSON? Not every allocation will have a username and password. 

Maybe add a `meta` column for arbitrary stuff?